tools/ocp2k8s

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+image=$1
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Get currently exported variables
+current_envs=$(env | grep -f <(cat ${SCRIPT_DIR}/../openshift/template.yaml | yq --raw-output '.parameters[] | "\(.name)"'))
```

Comment:
I'm still unsure what's going on in this script
Is it possible it will be more clear written with Python?

Reply:
That's very likely, as bash is not the most self-explanatory language, and python excels at that.
However I feel like this would be more complex in python, and will turn a ~5 line script into a reasonably sized script.

Infact I really wanted to avoid doing trickery to convert openshift manifest into k8s compatible ones, but I did not have a clear alternative. Thoughts?